tests/integration/mixer/envoymixer/logs/accesslog_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright Istio Authors
```

Comment:
Nit: can you change the dir path to tests/integration/mixer/envoy/logs/ ? the extra mixer in the path seems to be duplicated.